### PR TITLE
Feature/33 Add git flow warnings

### LINKED ISSUE #33: Enforce git flow branch naming contentions
`feature/*`, `hotfix/*`, `release/*`, `develop` or `master`

### PR DESCRIPTION
**Pull request by sean-clayton**
Closes #33

### CODE REVIEW

**Comment by decrapifier** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  The README has not been updated. Please update the README.
  </td>
    </tr>
  </tbody>
</table>


<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  Could not find any browser results.
  </td>
    </tr>
  
<tr>
      <td>:book:</td>
      <td>

  We were able to automatically fix some formatting issues in this PR for you!
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "bin": {
     "duti": "bin/duti"
   },
-  "version": "0.10.0",
+  "version": "0.10.1",
```
Comment (daedalus28) [inline package.json]: should be 0.11.0 right?

Comment (sean-clayton) [reply package.json]: d'oh, yup.

File: src/duties/pr.js [hunk 2]
```diff
@@ -63,11 +63,32 @@ let disallowedDescription = ({ danger, fail, config }) => {
...
+    'develop',
+    'feature/.*',
+    'hotfix/.*',
+    'release/.*',
+  ]
+  let branchName = _.get('github.pr.head.ref', danger)
+  let branchUsesGitFlow = _.some(
+    scheme => new RegExp(scheme).test(branchName),
+    gitFlowSchemes
+  )
```
Comment (daedalus28) [inline src/duties/pr.js]: Besides being shorter this also more efficient:

```js
let branchUsesGitFlow = F.testRegex(_.join('|', gitFlowSchemes))
```

**Pull request merged**